v2.0/src/AppRouter.js

```diff
@@ -292,24 +305,68 @@ class AppRouter extends Component {
       if (menu[4]) {
         newAbout.children = [...newAbout.children, menu.pop()];
       }
+      // remove menu items for pages which cadmins have selected as not enabled
+      newAbout.children = newAbout.children.filter((item) => {
```

Comment:
this variable Name sounds confusing to me.  Could you shed some more light on what it contains?

Reply:
I'm not sure what you're asking.  The changes I added take the menus which were modified, and eliminate the pages which were disabled.  newAbout is the About menu with one minor fix, and its four children can be enabled or disabled.